input/pagecontent/general-requirements.md

```diff
@@ -1,8 +1,8 @@
 ### Conforming to AU Core
 
 There are two different ways to implement AU Core:
-1. Profile Only Support: systems support only the resources as profiled by AU Core to represent digital health information.
-1. Profile Support + Interaction Support: systems support the resources as profiled by AU Core to represent digital health information (Profile Support) and [the RESTful FHIR API](http://hl7.org/fhir/R4/http.html) interactions defined for it (Interaction Support).
+1. [Profile Only Support](#profile-only-support): system that represent digital health information using AU Core profiles without the expectation to implement any FHIR specified interactions.
+1. [Profile Support + Interaction Support](#profile--interaction-support): system that represents digital health information using AU Core profiles (Profile Support) AND implement the [FHIR REST API](http://hl7.org/fhir/R4/http.html) interactions defined by an [AU Core Capability Statement](#au-core-capability-statements) (Interaction Support).
 
 The requirements of the FHIR standard and [FHIR Conformance Rules](http://hl7.org/fhir/conformance-rules.html) apply, and define the use of terms in this guide including the conformance verbs - SHALL, SHALL NOT, SHOULD, SHOULD NOT, MAY.
```

Comment:
Use the conformance verb formatting for conformance verbs for the listed conformance.

Reply:
Changed.